src/components/colorscale/color_attributes.js

```diff
@@ -19,7 +19,7 @@ module.exports = function makeColorScaleAttributes(context) {
             role: 'style',
             description: [
                 'Sets the ', context, ' color. It accepts either a specific color',
-                ' or an array of values that are mapped to the colorscale',
+                ' or an array of numbers that are mapped to the colorscale',
```

Comment:
Numbers or numeric strings to be more precise. Not sure if it's worth adding that amount of details in here though.